app/code/Magento/Config/Test/Unit/Model/ConfigTest.php

```diff
@@ -149,6 +158,41 @@ public function testSaveToCheckAdminSystemConfigChangedSectionEvent()
         $this->_model->save();
     }
 
+    public function testDoNotSaveReadOnlyFields()
```

Comment:
Why content of this method is different in original PR?

Reply:
Hi @ihor-sviziev 
Thank you for review 
In Magento 2.2 exist two additional calls of method `Magento\Config\Model\Config\Structure\Element\Group::getElement` (check https://github.com/phoenix128/magento2/blob/af78c2fe5a02e5f521f1c984f303063544ef71f6/app/code/Magento/Config/Model/Config.php line 229 and 231) unlike Magento 2.3 implementation. 
So in M2.3, we expect this method being invoked once as I've implemented.
$this->_configStructure->expects($this->once())
            ->method('getElement')
            ->with('section/1')
            ->will($this->returnValue($group));

Reply:
I believe this the only difference.